tilequeue/command.py

```diff
@@ -978,18 +978,23 @@ def tilequeue_prune_tiles_of_interest(cfg, peripherals):
     with psycopg2.connect(redshift_uri) as conn:
         with conn.cursor() as cur:
             cur.execute("""
-                select x, y, z
+                select x, y, z, tilesize
                 from tile_traffic_v4
                 where (date >= dateadd(day, -{days}, current_date))
-                  and (z between 10 and 16)
+                  and (z between 0 and 15)
```

Comment:
I made this change because I didn't want to assume we always wanted to have zooms 0 through 9 in the TOI. In my sample configuration I have the whole Earth at z0-z9 as an "immortal" area, so it gets added to the TOI that way.